.tool/lint

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+for d in $(find . -type d -not -iwholename '*.git*' -a -not -iname '.tool' -a -not -iwholename '*vendor*'); do
+	${GOPATH}/bin/gometalinter \
```

Comment:
This does not work when GOPATH contains multiple entries.

I think it is ok to assume PATH is set to contain gometalinter